extensions/sql-database-projects/src/common/uiConstants.ts

```diff
@@ -11,12 +11,15 @@ export namespace cssStyles {
 	export const fontWeightBold = { 'font-weight': 'bold' };
 	export const titleFontSize = 13;
 
-	export const labelWidth = '205px';
-	export const textboxWidth = '190px';
+	export const publishDialogLabelWidth = '205px';
+	export const publishDialogTextboxWidth = '190px';
 
 	export const addDatabaseReferenceDialogLabelWidth = '215px';
 	export const addDatabaseReferenceInputboxWidth = '220px';
 
+	export const createProjectFromDatabaseLabelWidth = '110px';
+	export const createProjectFromDatabaseTextboxWidth = '320px';
```

Comment:
this textbox looks a little bit wider than it needs to be and wider than the schema compare dialog's if that's what was trying to be matched.

Reply:
We were trying to match it to Connections dialog, instead of the SC dialog. In fact, because of the icons at the end of the textboxes, it had to be cut short a bit.